packages/wrangler/scripts/bundle.ts

```diff
@@ -54,19 +54,35 @@ async function buildMiniflareCLI(flags: BuildFlags = {}) {
 	});
 }
 
+async function buildCLIEntryPoint(flags: BuildFlags = {}) {
+	await build({
+		entryPoints: ["./bin-src/wrangler.js"],
+		outfile: "./bin/wrangler.js",
+		platform: "node",
+		format: "cjs",
+		target: "node12",
```

Comment:
I assume we're picking node12 since it's the most recent one to go EOL?

Reply:
just because it was the one referenced in the original issue, but I could go earlier too. lemme choose node10.